### DISCUSSION TITLE
The choice of the name "Lite" and the communication surrounding it create unnecessary hassle for everyone that could be avoided.

### DISCUSSION THREAD

**Discussion by romanr** [uBlockOrigin/uBlock-issues#3435] [General]
First of all, this is a fantastic extension, and I am very grateful for it. The choice of name and the README is probably the last thing on the developer's mind, but consider what an average user has to go through:

1. Encounter the warning in Chrome about the soon-to-be-deprecated extension.  
2. Click "Find alternatives."  
3. It opens the Store with "UBOL." I can't really trust it, knowing how many copycats there are. How can I be sure to find the real one?  
4. Go back to the old extension, click on configuration, then on that page click about, and click Source Code. (The message that UBOL is the replacement could be on the About page.)  
5. It opens GitHub; scroll through the README, but there is no mention of UBOL.  
6. Go to Issues, browse, and find nothing…  
7. Oh, there's an issue at the top that says, "This is a deprecated repository!"  
8. Okay, click on the link to go to the new issue tracker.  
9. Go to Issues. Browse the issues; finally, in https://github.com/uBlockOrigin/uBlock-issues/issues/3309, the question is addressed, but why "Lite"? That's another question discussed below.  

It would not be necessary for people to ask the same thing repeatedly if it were clearly explained in a README or UBO itself.  
The unfortunate choice of the suffix "Lite" historically implies a product with intentionally reduced functionality. Who knows how many people have thought it was time to switch to ABP or something similar because this is now "Lite"?  
It sounds as though UBOL is the "UBO MW3 edition."  
Leaving it as "Lite" will only inundate the tracker and discussions with questions from those who did not spend time exploring these details in the issues.  

If any assistance is needed with the copy or messaging regarding this, I would be happy to help.

**Comment by uBlock-user** [comment]
> The unfortunate choice of the suffix "Lite" historically implies a product with intentionally reduced functionality. 

Yes, it is exactly that. Hence the added suffix.

**Reply by garry-ut99** [reply]
I agree with the answer.

Besides:

> romanr:  The choice of the name "Lite" The choice of name, but why "Lite"? The unfortunate choice of the suffix "Lite" It sounds as though UBOL is the "UBO MW3 edition." Leaving it as "Lite" 

you are too late for discussion about the name, as it was already discussed and decided, hence your question regarding the name is a duplicate of:
- https://github.com/uBlockOrigin/uBlock-issues/discussions/2261

**Comment by gorhill** [comment]
> It opens the Store with "UBOL." I can't really trust it, knowing how many copycats there are. How can I be sure to find the real one?

In Details section, it shows:

![image](https://github.com/user-attachments/assets/b5347928-0bbd-44de-814a-b541940261d9)

Same author/email as uBO:

![image](https://github.com/user-attachments/assets/f057a21d-cba5-4b42-818f-8950be583c94)
